api/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "start": "nodemon server.js",
-    "tunnel": "autossh -M 0 -o \"ServerAliveInterval 30\" -o \"ServerAliveCountMax 3\" -R 80:localhost:3000 serveo.net",
+    "tunnel-mac-linux": "export $(cat .env | xargs) && npx localtunnel  --port 3000 --subdomain $API_TUNNEL_SUBDOMAIN",
```

Comment:
The extra code at the beginning of these two lines grabs the `api/.env` file variables in order to pass `API_TUNNEL_SUBDOMAIN` to localtunnel, so that you can get a consistent tunnelled URL that doesn't change